packages/platforms/accton/x86-64/as7535-28xb/onlp/builds/x86_64_accton_as7535_28xb/module/src/sfpi.c

```diff
@@ -371,6 +404,27 @@ onlp_sfpi_control_get(int port, onlp_sfp_control_t control, int* value)
 
         return ONLP_STATUS_OK;
     }
+    case ONLP_SFP_CONTROL_SOFT_RATE_SELECT: {
+        VALIDATE_SFP(port);
+
+        present = onlp_sfpi_is_present(port);
+        if(present == 1) {
+            mbit_identifier = onlp_sfpi_dev_readb(port, MBIT_IDENTIFIER_ADDR, MBIT_OFFSET);
+            if(mbit_identifier == RATE_SELECT) {
+                multirate = onlp_sfpi_dev_readb(port, PORT_EEPROM_DEVADDR, MULTIRATE_OFFSET);
+                *value = multirate;
```

Comment:
Please help validate the return value 'multirate' before assignment.

Reply:
addressed